test.rb

```diff
@@ -1,2 +1,13 @@
 puts 'Hello World'
 puts "hollow world"
+puts "More Badly formatted" +  "Code to flag"  + "rubocop Warnings"
+def test_func (test_input)
+  puts "This is a really, really, long line that should cause additional warnings from tools like reek"
```

Comment:
C: Prefer single-quoted strings when you don't need string interpolation or special symbols.